### ISSUE TITLE
 link error after 0.12.4 upgrade

### ISSUE THREAD

**Issue by dheijl** [MoAlyousef/fltk-rs#408]
Hi,

Windows 10, latest stable rust:

I upgraded to fltk-rs 0.12.4, ignored the "deprecated" warning about `app::set_color()`, and got the folllowing linker unresolved externals errors (Fl_foreground, Fl_background, Fl_background2): 

```shell
   = note: libfltk-1b27cc8207bdafa1.rlib(fltk-1b27cc8207bdafa1.fltk.eo5hwp80-cgu.9.rcgu.o) : error LNK2019: unresolved external symbol Fl_foreground referenced in function _ZN4fltk3app10foreground17h00085381559b55efE
          libfltk-1b27cc8207bdafa1.rlib(fltk-1b27cc8207bdafa1.fltk.eo5hwp80-cgu.9.rcgu.o) : error LNK2019: unresolved external symbol Fl_background referenced in function _ZN4fltk3app10background17heb07b03a2fc81ec2E
          libfltk-1b27cc8207bdafa1.rlib(fltk-1b27cc8207bdafa1.fltk.eo5hwp80-cgu.9.rcgu.o) : error LNK2019: unresolved external symbol Fl_background2 referenced in function _ZN4fltk3app11background217h508911371a6fbac0E
          C:\Users\Danny\source\rust\projects\swyh-rs\target\debug\deps\swyh_rs.exe : fatal error LNK1120: 3 unresolved externals
```

Reverting Cargo.toml and Cargo.lock to 0.12.3 fixed it.

**Comment by MoAlyousef** [comment]
Hi

It seems I messed up the publication of the crate. I'll yank version 0.12.4 and publish 0.12.5.

**Comment by MoAlyousef** [comment]
Can you try again. Hopefully this is fixed in version 0.12.5. 
Also thanks for trying out newer versions of fltk-rs :)

**Comment by dheijl** [comment]
Fixed with 0.12.5, thanks!

> Also thanks for trying out newer versions of fltk-rs :)

At my age one likes to live on the bleeding edge!
